app/views/cases/filters/index.html.slim

```diff
@@ -14,18 +14,14 @@
 = render partial: 'cases/shared/search_bar'
 
 = render partial: 'cases/shared/case_tabs'
-
 section.govuk-tabs__panel
-
-  - if request.path.include? '/cases/open'
+  - if !@query.available_filters(current_user, @current_tab_name).empty?
```

Comment:
I think we could loose the negation here if we instead used `.present?`. I find double negatives hard to parse.